package.json

```
@@ -158,7 +158,7 @@
   "scripts": {
     "lint": "npm-run-all -p -c lint-js lint-css",
```

```suggestion
    "lint": "npm-run-all -p -c lint-js lint-css lint-ts",
```
To enable CI for TS files. See:
https://github.com/DevExpress/DevExtreme/blob/988321cdedd1a9fc1a22ad598ca8eb101338b36a/docker-ci.sh#L10-L13